models/cice/model_mod.f90

```diff
@@ -2684,7 +2530,7 @@ subroutine use_default_state_variables( state_variables )
 character(len=*),  intent(inout) :: state_variables(:)
 
 ! strings must all be the same length for the gnu compiler
-state_variables( 1:5*num_state_table_columns ) = &
+state_variables( 1:5*3 ) = &
```

Comment:
```suggestion
state_variables( 1:15 ) = &
```

Reply:
Putting this change in the commit with other fixes to the POP and CICE model_mods